packages/calcite-components/src/components/list/list.tsx

```diff
@@ -800,9 +800,9 @@ export class List implements InteractiveComponent, LoadableComponent, SortableCo
     this.connectObserver();
 
     this.calciteListOrderChange.emit({
-      dragEl: sortItem,
-      fromEl: parentEl,
-      toEl: parentEl,
+      dragEl: sortItem as HTMLCalciteListItemElement,
```

Comment:
Do you need to type cast here? I think all of the elements are properly typed to what the interface expects.